TYPO3.Neos/Classes/TYPO3/Neos/Domain/Strategy/AssetUsageInNodePropertiesStrategy.php

```diff
@@ -88,15 +90,19 @@ public function getUsageReferences(AssetInterface $asset)
             return $this->firstlevelCache[$assetIdentifier];
         }
 
-        $userWorkspace = $this->userService->getPersonalWorkspace();
+        $userWorkspaceName = $this->userService->getPersonalWorkspace()->getName();
 
         $relatedNodes = [];
         foreach ($this->getRelatedNodes($asset) as $relatedNodeData) {
+            /** @var NodeData $relatedNodeData */
+            if ($relatedNodeData->isInternal()) {
+                continue;
+            }
             $accessible = $this->domainUserService->currentUserCanReadWorkspace($relatedNodeData->getWorkspace());
             if ($accessible) {
-                $context = $this->createContextMatchingNodeData($relatedNodeData);
+                $context = $this->createContentContext($userWorkspaceName);
             } else {
-                $context = $this->createContentContext($userWorkspace->getName());
+                $context = $this->createContextMatchingNodeData($relatedNodeData);
```

Comment:
You're right. I was assuming the own workspace is on top of the node workspace (i.e. user/live).
I'll change that.